### PR TITLE
Add Windows profile in Maven to use specific swagger-cli executable

### LINKED ISSUE #2826: Build failing on Windows machine
**Describe the bug**
The build of latest `develop` fails when running `mvn clean install -DskipTests -Pconsole,docker` command on Windows. 
**To Reproduce**
1. Checkout a new branch, up-to-date with the latest `develop`
2. Run `mvn clean install -DskipTests -Pconsole,docker` 
3. The build fails with **The system cannot find the file specified** error in directory `kapua\assembly\api`

**Expected behavior**
The build should pass without errors.

**Screenshots**
_Screenshot 1:_
image

_Screenshot 2_
image

**Version of Kapua**
1.1.0

**Type of deployment**
[ ] Local Vagrant deployment
[X] Docker
[ ] Openshift (in its variants) 
[ ] Others

**Main component affected**
[ ] Console (in case of console please report info on which browser you encountered the problem)
[ ] REST API
[ ] Message Broker
[X] - Others

**Additional context**
Discussion about this error was started on gitter, and as agreed with @lorthirk a new issue is opened.

### PR DESCRIPTION
**Pull request by lorthirk**
After merging #2795, [swagger-cli](https://www.npmjs.com/package/swagger-cli) is used to bundle all the OpenAPI files in a single file to be served from SwaggerUI. Apparently, when building under Windows, swagger-cli is installed in a different location than when building under macOS and Linux, so a new Maven profile, automatically activated when building in Windows, has been created to customize the swagger-cli executable path.

**Related Issue**
Fixes #2826

### CODE REVIEW

**Comment by lorthirk** [comment]
@ct-ajovanovic please test this fix, and let me know if you can safely build again.

Let's wait for this feedback before merging.

**Comment by code-incubus** [comment]
Hi @lorthirk, because ALeksadra is on holiday, I tried this instead of her on my Windows machine and it works fine, so I guess we can merge this.

**Review by Coduz — APPROVED** [review]

**Pull request merged**
